learn/react-native/performance-measurment.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Performance Measurement"
```

Comment:
Performance Measurement of React Native Mobile apps

Reply:
updated